### ISSUE TITLE
CPM: Identify how and where we want to robot to approach the control panel

### ISSUE THREAD

**Issue by rbtgagnon** [frc-88/2020-Robot#2]
If we approach the control panel from the side we may not be in a save position and our ability to identify our current detected color and the color under the sensor may be more complicated. 
- if the driver is controlling our ability to find a color, this may be the easier position for that strategy

If we approach from the trench me may be more protected and have more landmarks that will enable us to identify how to translate the current color we see to the current color that is under the sensor. (off by 2?)

**Comment by nix342** [comment]
I don't think that it is possible to manipulate the control panel from the side. One of the supports for the shield generator structure is located there. Unless you mean just approaching from the side, but driving into the trench run and deploying the CPM from the side of the robot. I think there are drawbacks to either idea. lining up to the target while driving sideways (left-to-right or right-to-left from the driver's perspective) is difficult. There are a number of issues I can see with deploying the CPM to the side.

I've been thinking about this a bit and I think that the optimal location for our robot would be on the intake side of the robot. As a shooting robot, we will most often be in the long side of the trench and it makes sense to me to be able to manipulate the wheel on our way back through the tunnel just after we have hit the target needed for the stage.
